docs/chapters/workflow/release-process.rst

```diff
@@ -0,0 +1,63 @@
+Release process
+===============
+
+Component attributes
+--------------------
+To properly map a release of the PELUX Baseline, we collect a number of
+attributes for the components that make up a release. This goes further than
+only the artifacts produced in the release, since an artifact can be an
+aggregate of several components (such as yocto layers).
+
+Versioning
+    The PELUX Baseline uses a release versioning scheme similar to what Debian
+    uses for their releases. A version number is X.Y where X and Y are integers.
+    Most releases will be X.0, with bug fixes and security updates in subsequent
+    X.Y releases. Any more substantial development done after a release will go
+    into the next release.
+
+Git tags
+    Wherever we use git tags below, the tag name should map to the version name.
+    This can be done by tagging it as ``PELUX_<Version>`` or similar, depending
+    on what repository it is.
+
+Yocto manifests
+^^^^^^^^^^^^^^^
+The yocto manifest has the following attributes:
+* Branch name
+* Git tag
+
+The branch name is generic and should map to a yocto release branch name,
+whereas the git tag is specific and maps to a specific commit on that branch.
+
+The manifest should point out all layers it uses by pointing to a specific git
+commit.
```

Comment:
This little piece of info seems a bit off topic. Feels more like some step of creating a release manifest (as in content, not properties/attributes), which I assume we need to state somewhere else as well (making this a duplicate info)

Reply:
I would say it's fine as long as you rationalize why you wouldn't use a git tag but a commit instead. For the purpose of a release.

Reply:
I both agree and disagree. Yes, this is not an attribute of the manifest itself, but on the other hand, pointing to specific commits in the manifest is an important property of a release, for example for reproducability reasons.

I would vote for keeping it in but (re)moving it if and when we state it elsewhere.

Reply:
And yes, the same work that's been done to produce this list should be defined in the Blueprint, I completely agree.

Reply:
I think this should be a text about the attributes only.

The concept of creating a "release manifest" and the rationale and details about that will need to be added to the SWF as part of the ongoing release work anyway. So I personally don't see a reason to put it in a place that we know will be wrong later, but as long as this doesn't end up as cruft then it doesn't matter that much now.

Reply:
Ok, so we want the attribute list more to be a what and not a how? I can live with that, and I guess we will link to the how-article from this article anyway.

Reply:
Yeah I think that's best. Even if we later add the how to here anyway, I'd still think that the content of the manifest will be a separate thing still.